Fix: 메인페이지 / Welcome 페이지 리팩토링

Artlogy opened the pull request:
[필요한 사항]
- head.html 적용
- topbar.html 적용

1) [리스트화면] topbar.html 이용하셔서 판매하기/구매하기에 따라 분류 적용
2) [리스트화면] 장르별로 카테고리 분류 적용
3) 최근 책 등록 순서대로 [리스트화면]에 랜더링 적용

wkdgusdnr55-cloud committed:
Fix: 메인페이지 / Welcome 페이지 리팩토링

[필요한 사항]
- head.html 적용
- topbar.html 적용

1) [리스트화면] topbar.html 이용하셔서 판매하기/구매하기에 따라 분류 적용
2) [리스트화면] 장르별로 카테고리 분류 적용
3) 최근 책 등록 순서대로 [리스트화면]에 랜더링 적용

blingblin-g commented on app/controller/home.py:
```diff
@@ -20,16 +20,16 @@ def welcome():
     db = get_db()
+    print(f'>>>>>>>>>{request.json}')
     request_data = dict(request.json);
-    print(request_data)
-    if request_data.get('category') is not None:
+    print(f'>>>>>>>>>{request_data}')
```
`print()`는 지워주시는 게 좋을 것 같아요

blingblin-g commented on app/templates/mainpage.html:
```diff
@@ -1,76 +1,94 @@
+        return params;
+    }
+
+    async function getList(genre = null, is_selling = true) {
+        const data = await api('/api/getList', 'POST', {genre: genre, is_selling: is_selling});
+        const contentBox = document.getElementById('content-box')
+        while (contentBox.hasChildNodes()) {
+            contentBox.removeChild(contentBox.firstChild)
+        }
+        let cardboard = '';
+        for (let i = 0; i < data.length; i++) {
+            cardboard = `<div class="card" num=${data[i].num} onclick="window.location.href='post/${data[i].num}'">
+                                        <img src = ${data[i].img} alt="이미지 링크 확인"/>
```
jQuery보다 바닐라 JS로 구현하신 게 좋네요

blingblin-g reviewed (approved):
고생하셨습니다. `print()`만 지워주세요

blingblin-g commented on app/templates/navbar.html:
```diff
@@ -14,7 +14,7 @@
             </form>
             {% else %}
             <ul class="navbar_login">
-                <li><a class="login" href="">로그인</a></li>
+                <li><a class="login" href="/account/signin">로그인</a></li>
             </ul>
```
위의 로그아웃 버튼과 전혀 다른 방식의 코드군요 👀

marksenee committed:
Fix: /list로 navigation 연결 및 is_selling type 변경

marksenee committed:
Change: [css], [html] 디렉토리 변경

blingblin-g committed:
Merge pull request #16 from innovation-camp/fix/navigation

Fix: /list로 navigation 연결 및 is_selling type 변경

wkdgusdnr55-cloud committed:
Fix: 메인페이지 / Welcome 페이지 리팩토링

[필요한 사항]
- head.html 적용
- topbar.html 적용

1) [리스트화면] topbar.html 이용하셔서 판매하기/구매하기에 따라 분류 적용
2) [리스트화면] 장르별로 카테고리 분류 적용
3) 최근 책 등록 순서대로 [리스트화면]에 랜더링 적용

wkdgusdnr55-cloud committed:
Merge branch 'feature/main' of github.com:innovation-camp/heonchaegjulge into feature/main

wkdgusdnr55-cloud committed:
Fix: 메인페이지 / Welcome 페이지 리팩토링

[필요한 사항]
- head.html 적용
- topbar.html 적용

1) [리스트화면] topbar.html 이용하셔서 판매하기/구매하기에 따라 분류 적용
2) [리스트화면] 장르별로 카테고리 분류 적용
3) 최근 책 등록 순서대로 [리스트화면]에 랜더링 적용

marksenee reviewed (approved):
확인했습니다!

wkdgusdnr55-cloud committed:
Fix: 메인페이지 / Welcome 페이지 리팩토링

[필요한 사항]
- head.html 적용
- topbar.html 적용

1) [리스트화면] topbar.html 이용하셔서 판매하기/구매하기에 따라 분류 적용
2) [리스트화면] 장르별로 카테고리 분류 적용
3) 최근 책 등록 순서대로 [리스트화면]에 랜더링 적용
